src/OrchardCore.Modules/OrchardCore.Spatial/Assets/js/coordinate-leaflet-field.js

```diff
@@ -0,0 +1,43 @@
+function addMapPicker() {
+
+    var lat = $('[data-latitude]').val();
+    var long = $('[data-longitude]').val();
+
+    var mapCenter = [-34.92879716252353, 138.59996905763066];
```

Comment:
This is not appropriate it should be configurable at least. Now this points to Adelaide Australia by default when there is no Lat / Long values.

Reply:
Resolved by using https://en.wikipedia.org/wiki/Geographical_centre_of_Earth

In 2016, Google Maps marked Isenberg's result of 40°52′N 34°34′ECoordinates: 40°52′N 34°34′E as the geographical center of Earth.

https://geohack.toolforge.org/geohack.php?pagename=Geographical_centre_of_Earth&params=40_52_N_34_34_E_type:landmark_scale:1000000&title=Turkey%E2%80%99s+%C3%87orum+as+center+of+the+Earth+%28Google+2016%29